### ISSUE TITLE
Test case: Allergy

### ISSUE THREAD

**Issue by jkiddo** [Gravitate-Health/focusing-interfaces#2]
Given the drug of http://build.fhir.org/ig/HL7/vulcan-eproduct-info/Bundle-bundlepackageleaflet75.html and the IPS of https://build.fhir.org/ig/hl7-eu/gravitate-health-ips/Bundle-gravitate-helen.json.html - GLens focus should spot the lactose intollerance and highlight that in the returned ePI.

**Comment by jkiddo** [comment]
@cander2 I don't see that the ePI bundle contains any structured information about Karvea containing lactose. That is needed. Will you look at that?

**Comment by cander2** [comment]
Yes. See Karvea example. ePI in one folder and SPOR aspects in separate folder.

Given EMA's approach, no direct connect from Composition to ingredient. Have to go ePI Bundle > Medicinal Product > Administrable Product/Manufactured Item > Ingredient

https://github.com/HL7/vulcan-eproduct-info/blob/master/input/fsh/examples/karvea-ema/spor/ingredient-lactosemonohydrate.fsh

Does this help?

**Comment by jkiddo** [comment]
@cander2 The [Karvea](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/Bundle-bundlepackageleaflet75.html) contains a [list](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/List-listofproducts.html) with a reference to the [MedicinalProductDefinition](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/MedicinalProductDefinition-karvea75mgblisterx28.html). 
MedicinalProductDefinition is referenced from the [`AdministrableProduct.fromOf`](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/AdministrableProductDefinition-adminproductkarvea75.html) and `AdministrableProduct. producedFrom`references the [ManufacturedItemDefinition](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/ManufacturedItemDefinition-manitemkarvea75.html). The [ManufacturedItemDefinition](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/ManufacturedItemDefinition-manitemkarvea75.html) is referenced from the [Ingredient](http://build.fhir.org/ig/HL7/vulcan-eproduct-info/Ingredient-lactosemonohydrate.html) using `Ingredient.for`.

Correct? Which part of the linked graph does not exist in the EMA approach. The one between the list and MedicinalProductDefinition - or is it 'just' a symbolic link? Some link must exist.

**Comment by cander2** [comment]
Correct. 

A bit sketchy on how SPOR works in detail but I believe the graph you noted above is in scope of what EMA is doing.

There is a direct link between the ePI List (for products) and the MedicinalProductDefinition

**Comment by jkiddo** [comment]
So where is the missing link @cander2 ?

**Comment by cander2** [comment]
Is there a missing link? I thought the graph described above is all we need. Or is that not the case?

**Comment by jkiddo** [comment]
@cander2 in the graph I mentioned, the entire graph is there, but that will not be the case for EMA, correct? 

If I look at https://github.com/EuropeanMedicinesAgency/EU-ePI-common-standard/blob/c68a60a7fcfc4288d4d01ab3b51b437d4419dd73/XML%20templates/ePI_template_instance.xml#L297 - EMA have put a literal reference to the MedicinalProductDefinition but is that expected to resolve? or is it expected to only resolve for some special parties?

**Comment by cander2** [comment]
I'm guessing no and we will have to do two separate searches to get there. But let me reach out to Rik to confirm.

**Comment by jkiddo** [comment]
I already did that
